infermedica_api/webservice.py

```diff
@@ -182,11 +182,11 @@ def search(self, phrase, sex=None, max_results=8, filters=None, **kwargs):
         :rtype: list
         """
         method = self.__get_method('search')
-        params = {
+        params = kwargs.pop('params', kwargs)
```

Comment:
I'm not sure if it good approach...

Reply:
Imo no, we don't know what will be put to kwargs. `params = kwargs.pop('params', {})` should be the way to go.

Reply:
@akszydelko done